package.json

```diff
@@ -30,15 +30,16 @@
     "jquery": "^3.6.0",
     "leaflet": "^1.9.3",
     "lodash": "^4.17.21",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-leaflet": "^3.2.5",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-leaflet": "^4.2.1",
     "react-modal": "^3.16.1"
   },
   "devDependencies": {
     "@babel/core": "^7.24.6",
     "@babel/eslint-parser": "^7.24.1",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-transform-modules-commonjs": "^7.24.7",
```

Comment:
This should already be included by `@babel/preset-env` so doesn't need to be listed separately